### PR TITLE
tools/genStrSession: Don't crash when generating sessions for UserIsBot

### PR DESCRIPTION
**Pull request by TomyPrs**

### CODE REVIEW

File: tools/genStrSession.py [hunk 1]
```diff
@@ -25,9 +26,14 @@ async def genStrSession() -> None:  # pylint: disable=missing-function-docstring
             api_hash=os.environ.get("API_HASH") or input("Enter Telegram API HASH: ")
     ) as userge:
         print("\nprocessing...")
-        await userge.send_message(
+        doneStr = "sent to saved messages!"
+        try:
+            await userge.send_message(
             "me", f"#USERGE #HU_STRING_SESSION\n\n```{await userge.export_session_string()}```")
-        print("Done !, session string has been sent to saved messages!")
+        except UserIsBot:
```
Comment (Krishna-Singhal) [inline tools/genStrSession.py:34]: when this exception will raise ? can you explain

Comment (TomyPrs) [reply tools/genStrSession.py:34]: When you generate bot token, title tell anything :3

File: tools/genStrSession.py [hunk 2]
```diff
@@ -25,9 +26,14 @@ async def genStrSession() -> None:  # pylint: disable=missing-function-docstring
...
+            doneStr = "has been successfully printed!"
```
Comment (Harsh-br0) [inline tools/genStrSession.py:35]: Sup @TomyPrs 

```suggestion
            doneStr = "successfully printed!"
```

Cuz its already getting printed in line 36

File: tools/genStrSession.py (hunk 1 above)
Comment (Krishna-Singhal) [reply tools/genStrSession.py:34]: > When you generate bot token, title tell anything :3

you wrote "when generating a bot" so i didnt understood with this

**Pull request merged**